src/views/main/index.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useCallback, useEffect } from 'react'
+import ClipLoader from 'react-spinners/ClipLoader'
+import { useSelector, useDispatch } from 'react-redux'
+
+import Header from '_components/Header'
+import { getUser, updateUser } from '_modules/user/actions'
+import { getUserSelector, loadingUpdateUser, loadingUser } from '_modules/user/selectors'
+
+import styles from './styles.css'
+
+const ALLOW_IMAGES_EXTENSIONS = ['.jpeg', '.jpg', '.png']
+function getBase64(file) {
+  return new Promise((resolve, reject) => {
+    const reader = new FileReader()
+    reader.readAsDataURL(file)
+    reader.onload = () => resolve(reader.result)
+    reader.onerror = error => reject(error)
+  })
+}
+
+const Main = () => {
+  const user = useSelector(getUserSelector)
+  const dispatch = useDispatch()
+  const loading = useSelector(loadingUser)
+  const loadingUpdateName = useSelector(loadingUpdateUser)
+
+  const handleSubmit = useCallback(
+    evt => {
```

Comment:
```suggestion
    event => {
```

No need to reduce the size of words